Kempe/Kempe.txt

```diff
@@ -0,0 +1,145 @@
+
+This code is contributed by Riyazul555
+
+
+Time Complexity = O(N^2)
+Space Complexity = O(N)
+
+
+This is the PSEUDOCODE of Kempe's Algrithm 
+
+
+
+
+procedure BFS(nod, G, viz)
+    Q = Queue()
+    viz[nod] = true
```

Comment:
We use "<-" for assignment and capitalize Boolean variables. Please fix where applicable.